protocol/messenger_identity.go

```diff
@@ -150,6 +150,8 @@ func (m *Messenger) setInstallationHostname() error {
 			if err != nil {
 				return err
 			}
+			// REVIEW(ilmotta) imd.Name is always empty in this else branch, so why do
+			// we concatenate it?
 			imd.Name = fmt.Sprintf("%s %s", hn, imd.Name)
```

Comment:
Thanks @qfrank, that helped me now reach the root commit.

It looks like this tiny problem was introduced by commit https://github.com/status-im/status-go/commit/f87a38074#diff-b0493cab0297f2e6f0866d54b819f48ff079659dd62f68648743b32819ecbbcfR166-R174

@Samyoul, this commit is quite old already, so it's totally expected if you don't remember, but do you know why we are always concatenating `imd.Name` when it's in fact empty? This implementation carried over and it's still present in `develop`. The `Sprintf` line always generates a metadata name with a whitespace at the end, which I caught by chance while unit testing installation initialization.

It's probably harmless, otherwise it would have caused problems.